schemas/HPXMLDataTypes.xsd

```diff
@@ -1663,6 +1664,13 @@
 			<xs:enumeration value="other"/>
 		</xs:restriction>
 	</xs:simpleType>
+	<xs:simpleType name="AtticWallType">
+		<xs:restriction base="xs:string">
+			<xs:enumeration value="gable"/>
+			<xs:enumeration value="knee wall"/>
+			<xs:enumeration value="other"/>
+		</xs:restriction>
```

Comment:
For foundation walls and above-grade walls, the WallType enumerations are things like WoodStud, DoubleWoodStud, DoubleBrick, Solid Concrete, etc,. It'd be nice to have similar options here too. I agree with @GamalielL that the *AdjacentTo elements would be a preferable way of designating whether it's a gable, knee wall, etc.